test/lib/analyzer/image-inspector.test.ts

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node_modules/.bin/ts-node
+// Shebang is required, and file *has* to be executable: chmod +x file.test.js
+// See: https://github.com/tapjs/node-tap/issues/313#issuecomment-250067741
+
+// tslint:disable:max-line-length
+// tslint:disable:object-literal-key-quotes
```

Comment:
Don't disable linting please; lets just fix the errors, or disable for specific blocks, not whole files :smile:

Reply:
didn't touch this code :)
It looks new because of the renaming...

But I'll fix it anyway

Reply:
Ah, couldn't tell. Leave it then, I thought it was new cause of the diff.